### PR TITLE
fix(scylla-bench): Add iteration parameter to guarantees the load will run during all test

### PR DESCRIPTION
**Pull request by juliayakovlev**
'-duration' flag in scylla-bench command is actually timeout. Duration doesn't not guarantees that
 the command will run duration time. It guarantees that if the test will continue to run when
duration time is achieved, command proceeding will be interrupted. To guarantees that the load
will run during all test the '-iteration' parameter need to be added to the command.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-test_duration: 6480
+test_duration: 30
```
Comment (roydahan) [inline test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:1]: ??

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml [hunk 2]
```diff
@@ -46,10 +46,10 @@ stress_cmd: [
...
-                  "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -rows-per-request=10 -consistency-level=quorum -timeout=90s -partition-offset=1001 -concurrency=100 -connection-count=100 -duration=6420m"
+                  "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -partition-offset=1001 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -iterations=1000 -timeout=90s",
+                  "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=250 -clustering-row-count=100000 -clustering-row-size=uniform:3072..5120 -concurrency=10 -connection-count=10 -rows-per-request=10 -consistency-level=quorum -iterations=1000 -timeout=300s -validate-data",
```
Comment (roydahan) [inline test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:50]: Is it intended?

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml (hunk 1 above)
Comment (juliayakovlev) [reply test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:1]: fixed

**Review by bentsi — APPROVED** [review]

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml (hunk 2 above)
Comment (juliayakovlev) [reply test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml]: Yes. I want to increase run time. I just changed the iteration numbers

**Review by roydahan — APPROVED** [review]

**Pull request merged**